src/helpers/documentation.js

```diff
@@ -376,8 +376,38 @@ export const groupByCategory = (referenceDocs) => {
   }, {});
 };
 
+/**
+ * consolidateToSection is a reducer function to consolidate
+ * all of subsequent items after <h2/> into an array of section
+ * For <TrackedContent/> to surround its H2 and the content for /docs
+ */
+export const consolidateToSection = () => {
```

Comment:
Ah so this is looking for the H2s, then marking all content between them as a "section" of content? Nice